Command/CertificateGenerateCommand.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace AcmePhp\Bundle\Command;
+
+use AcmePhp\Bundle\DomainConfiguration\DomainConfiguration;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CertificateGenerateCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('acme_php:generate')
```

Comment:
I don't like underscores in commands, I find it weird. What about using simply `acmephp`?